File: clients/python/examples/collections-insert-one-lexical.py
from astrapy import DataAPIClient

# Get an existing collection
client = DataAPIClient()
database = client.get_database(
    "**API_ENDPOINT**", token="**APPLICATION_TOKEN**"
)
collection = database.get_collection("**COLLECTION_NAME**")

# Insert a document
collection.insert_one(
    {
        "name": "Jane Doe",
        "$lexical": "An active hiker, runner, and triathlete who loves the outdoors.",
    },
)

File: clients/java/examples/collections-insert-one-lexical.java
import com.datastax.astra.client.DataAPIClient;
import com.datastax.astra.client.collections.Collection;
import com.datastax.astra.client.collections.definition.documents.Document;

public class Example {

  public static void main(String[] args) {
    // Get an existing collection
    Collection<Document> collection =
        new DataAPIClient("**APPLICATION_TOKEN**")
            .getDatabase("**API_ENDPOINT**")
            .getCollection("**COLLECTION_NAME**");

    Document document =
        new Document()
            .append("name", "Jane Doe")
            .append("$lexical", "An active hiker, runner, and triathlete who loves the outdoors.");

    collection.insertOne(document);
  }
}
